### ISSUE TITLE
Giving Exporting Error: "Project export for platform: [platform] - x Failed."

### ISSUE THREAD

**Issue by mixedissmiling** [godotengine/godot#64260]
### Godot version

3.5

### System information

Windows 11

### Issue description

When exporting my game I receive the error from above `Project export for platform: [platform] - Failed.` I've never had this issue before 3.5.

### Steps to reproduce

Go to export a game of any version with the 3.5 export templates. 
NOTE: unable to reproduce with different project.

### Minimal reproduction project

unable, the error only is in my project for some reason

**Comment by Calinou** [comment]
@mixedissmiling Please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot. We can't do anything without access to a project.

**Comment by Calinou** [comment]
> I can upload the entire original project file though.

Can you upload the entire project in this case? :slightly_smiling_face:

**Comment by mixedissmiling** [comment]
https://www.mediafire.com/file/n2y029fh42iblib/Rocket_Hamster.zip/file
File was too big to upload through github

**Comment by mixedissmiling** [comment]
@Calinou did you get the file okay?

**Comment by Calinou** [comment]
> @Calinou did you get the file okay?

Yes, but I can't look into each project individually due to lack of time. Other contributors may have a look eventually, but I can't make any guarantees.

**Comment by mixedissmiling** [comment]
Alright, just wanted to make sure

**Comment by mixedissmiling** [comment]
UPDATE: Turns out the issue comes from a DiscordRPC Plugin, it said it had an issue with one of the scripts but I didn't think anything of it. Deleting the addons folder fixed the issue

**Comment by akien-mga** [comment]
Closing as this was a plugin specific issue.
